### ISSUE TITLE
Search from CLI doesn't work

### ISSUE THREAD

**Issue by ondj** [nextcloud/files_fulltextsearch#72]
Example: 
```
# sudo -u nginx php /var/www/nextcloud/occ fulltextsearch:search admin help
search
An unhandled exception has been thrown:
TypeError: Argument 1 passed to OCA\Files_FullTextSearch\Service\FilesService::getFileFromId() must be of the type string, null given, called in /var/www/nextcloud/apps/files_fulltextsearch/lib/Service/SearchService.php on line 241 and defined in /var/www/nextcloud/apps/files_fulltextsearch/lib/Service/FilesService.php:432
Stack trace:
#0 /var/www/nextcloud/apps/files_fulltextsearch/lib/Service/SearchService.php(241): OCA\Files_FullTextSearch\Service\FilesService->getFileFromId(NULL, 8)
#1 /var/www/nextcloud/apps/files_fulltextsearch/lib/Service/SearchService.php(219): OCA\Files_FullTextSearch\Service\SearchService->setDocumentInfo(Object(OCA\Files_FullTextSearch\Model\FilesDocument))
```

App versions:
```
  - fulltextsearch: 1.3.4
  - fulltextsearch_elasticsearch: 1.3.3
  - files_fulltextsearch: 1.3.2
```

**Comment by robertschulze** [comment]
I have the same issue
`  - fulltextsearch: 1.3.6
  - fulltextsearch_elasticsearch: 1.3.5
  - files_fulltextsearch: 1.3.5`

**Comment by Happyfeet01** [comment]
i have the same Problem.

In Connection.php line 632:
                                                                                                                                                                                                                  
  {"error":{"root_cause":[{"type":"x_content_parse_exception","reason":"[1:63] [bool] failed to parse field [should]"}],"type":"x_content_parse_exception","reason":"[1:63] [bool] failed to parse field [must]"  
  ,"caused_by":{"type":"x_content_parse_exception","reason":"[1:63] [bool] failed to parse field [should]","caused_by":{"type":"x_content_parse_exception","reason":"[1:63] [bool] failed to parse field [should  
  ]","caused_by":{"type":"illegal_state_exception","reason":"expected value but got [START_ARRAY]"}}}},"status":400}

**Comment by TheNetStriker** [comment]
I also get the same error in Nextcloud 20 with all fultext apps on version 2.0.

**Comment by acsfer** [comment]
Can you please confirm this still hapens with both latest versions of Nexcloud and `files_fulltextsearch` apps?

**Comment by joshtrichards** [comment]
Fixed in #104
